src/cli/annotations.rs

```diff
@@ -291,7 +296,7 @@ impl Execute for Merge {
 
         writeln!(
             stdout,
-            "Successfully merged annotations from {} files to {}",
+            "Successfully merged annotations from {} files to {:?}",
```

Comment:
Missing a perdiod :-)
```suggestion
            "Successfully merged annotations from {} files to {:?}.",
```